src/components/filter/row/EventResolverRow.tsx

```diff
@@ -120,17 +138,27 @@ export default function EventResolverRow({ config }: { config: FilterRowEventRes
 					)}
 				</div>
 			)}
-			<div className={searchStatusIconClassname} />
-			<input
-				type='text'
+			{(isLoading || isError) && <div className={searchStatusIconClassname} />}
```

Comment:
After finding event by id we show event data: name, timestamp and status. But now if you find event by id and you focus input and then blur this data doesn't show up